akka-actor-typed-tests/src/test/scala/akka/actor/typed/javadsl/AdaptationFailureSpec.scala

```diff
@@ -85,8 +87,8 @@ class AdaptationFailureSpec extends ScalaTestWithActorTestKit with AnyWordSpecLi
             Behaviors.receiveSignal {
               case (_, Terminated(`ref`)) =>
                 probe.ref ! "actor-stopped"
-                Behaviors.same
-              case _ => Behaviors.unhandled
+                Behaviors.same()
+              case _ => Behaviors.unhandled()
```

Comment:
Hmm, `Behaviors.same` and `unhandled` are defined without param list, is that not supported in Scala 3?

Edit: aha, it's https://github.com/lampepfl/dotty/issues/10347 ?

Reply:
Yeah... I submitted a PR to fix that issue: https://github.com/lampepfl/dotty/pull/12710

We can either wait for a release with that fix. Or we can keep the definition of `same` / `unhandled` (and a few others) the way they were, and insert casts at the use-sites in Java sources for now.

I changed the definitions because they are in `javadsl`. For Java clients, this doesn't change anything (the empty parameter list was added anyway), but if there are Scala clients that use `javadsl`, it matters (as in the test here).

Reply:
I think this is fine since it is only about the Java DSL and Java users cannot anyway call without param list.